### DISCUSSION TITLE
Daily Standup - Monday, April 24

### DISCUSSION THREAD

**Discussion by antonieto** [antonieto/ibm-dashboard#142] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by Int-Angel** [comment]
**What did you achieve in the last 24 hours?**:
 - @antonieto and I deployed and configurate first version of DB with prisma in azure

**What are your priorities for the next 24 hours?**:
 - Prisma models and migrations

**Blockers**:
 - noe

**Shoutouts**:
 - @antonieto for explaining how docker works

**Comment by paotg123** [comment]
**What did you achieve in the last 24 hours?**:
 - design logo
 - see things about the log in

**What are your priorities for the next 24 hours?**:
 - same tasks

**Blockers**:
 -I need your opinion 

**Shoutouts**:
 - @username for @everyone

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - #135 
 - #137 with @Int-Angel 

**What are your priorities for the next 24 hours?**:
 - #136 
 - #143

**Blockers**:
 -

**Shoutouts**:
 - @Int-Angel 
 - @paotg123
